docs/concepts/person_sampling.md

```diff
@@ -1,240 +1,164 @@
-| neuroticism | dict | Neuroticism (t\_score, label, description) |
-
-Each personality trait contains:
-
-* `t_score`: Standardized score (typically 0-100)
-* `label`: Interpretive label ("low", "average", "high", "very high")
-* `description`: Detailed behavioral description
-
-#### Synthetic Persona Fields (Available when `with_synthetic_personas=True`)
-
-##### Background and Development
-
-| Field Name | Type | Description |
-| ----- | ----- | ----- |
-| cultural\_background | str | Detailed narrative about cultural influences and upbringing |
-| skills\_and\_expertise | str | Comprehensive description of professional and personal capabilities |
-| skills\_and\_expertise\_list | str | List format of key skills and competencies |
-| hobbies\_and\_interests | str | Detailed description of personal interests and activities |
-| hobbies\_and\_interests\_list | str | List format of hobbies and interests |
-| career\_goals\_and\_ambitions | str | Professional aspirations and long-term objectives |
-
-##### Persona Profiles
-
-| Field Name | Type | Description |
-| ----- | ----- | ----- |
-| persona | str | Brief summary personality profile |
-| detailed\_persona | str | Comprehensive personality and behavioral description |
-| professional\_persona | str | Work environment personality and career approach |
-| finance\_persona | str | Financial decision-making style and money management approach |
-| healthcare\_persona | str | Health and wellness attitudes and behaviors |
-| sports\_persona | str | Sports interests and physical activity preferences |
-| arts\_persona | str | Artistic tastes, cultural interests, and creative preferences |
-| travel\_persona | str | Travel style, preferences, and exploration approach |
-| culinary\_persona | str | Food interests, cooking style, and dining preferences |
-
 ---
 
-## Best Practices
-
-### Choosing Configuration Options
-
-* **Use locales that are backed by a Nemotron-Personas dataset** for maximum demographic accuracy and realism
-* **Enable `with_synthetic_personas=True`** when you need rich character development, personalized content generation, or comprehensive behavioral modeling
-* **Disable synthetic personas** for basic demographic testing or when computational efficiency is prioritized
-
-### Effective Persona Usage
-
-* **Match persona depth to use case**: Use basic personas for simple applications, detailed personas for comprehensive character modeling
-* **Leverage context-specific personas**: Use `professional_persona` for workplace scenarios, `culinary_persona` for food-related applications
-* **Combine multiple persona fields** in prompts for richer, more nuanced content generation
+## Approach 2: Nemotron Personas Datasets
 
-### Performance Considerations
+### What It Does
+Uses curated Nemotron Personas datasets from NVIDIA GPU Cloud (NGC) to generate demographically accurate person data with rich personality profiles and behavioral characteristics.
```

Comment:
Would be good to clarify that these are extended versions of the open-source datasets.